### PR TITLE
News update for v1.11.0 rc2

### PR DESCRIPTION
**Pull request by shamisp**
## What
News update before release 

## How ?
Manual parsing of git log

### CODE REVIEW

File: NEWS [hunk 1]
```diff
@@ -7,11 +7,186 @@
 ##
 #
 
-## Current
+## 1.11.0 RC2 (TBD)
 ### Features:
+#### Core
+* Added support for UCX monitoring using virtual file system (VFS)/FUSE
+* Added support for applications with static libcuda linking
```
Comment (yosefe) [inline NEWS:14]: libcuda -> CUDA runtime

File: NEWS [hunk 2]
```diff
@@ -7,11 +7,186 @@
...
+* Added support for configuration files
```
Comment (yosefe) [inline NEWS:15]: files->file

File: NEWS [hunk 3]
```diff
@@ -7,11 +7,186 @@
...
+* Updated clang format configuration
+#### CI
```
Comment (yosefe) [inline NEWS:17]: let's move CI to the end?

File: NEWS [hunk 4]
```diff
@@ -7,11 +7,186 @@
...
+* Added Fedora 34, RHEL 7.2, 7.4
+* Added PGI support from HPC-SDK module
+* Added docker image with CUDA 11.2
+* Added IODEMO test
+* Added Ubuntu 20.4
+* Added test for connection manager fallback in client-server testing
+* Added loopback interface for tcp testing
 #### UCP
-* Added API for querying UCP library attributes
+* Added new API for active messages
```
Comment (yosefe) [inline NEWS:27]: new -> rendezvous

File: NEWS [hunk 5]
```diff
@@ -7,11 +7,186 @@
...
+* Added context name to UCP parameters
+* Added name to endpoint parameters
+* Added name to worker parameters
+* Added flag to silence request leak check
+* Added API for endpoint performance evaluation
+* Added API for multi-fragment eager messages
```
Comment (yosefe) [inline NEWS:33]: pls remove this

File: NEWS [hunk 6]
```diff
@@ -7,11 +7,186 @@
...
+* Added IODEMO test
+* Added Ubuntu 20.4
+* Added test for connection manager fallback in client-server testing
+* Added loopback interface for tcp testing
 #### UCP
-* Added API for querying UCP library attributes
+* Added new API for active messages
+* Added context name to UCP parameters
+* Added name to endpoint parameters
+* Added name to worker parameters
```
Comment (yosefe) [inline NEWS:30]: Added user-defined name to context, worker, and endpoint objects

File: NEWS [hunk 7]
```diff
@@ -7,11 +7,186 @@
...
+* Added flag to silence request leak check
+* Added API for endpoint performance evaluation
+* Added API for multi-fragment eager messages
+* Added API - ucp_request_query
+* Added API - ucp_lib_query
+* Ported connection manager to a new UCT API
+* Added bandwidth optimizations for new protocols multi-lane
+* Added support for multi-rail over lanes with BW ratio >= 1/4
+* Refactored request ID
```
Comment (yosefe) [inline NEWS:39]: Track outstanding requests and abort them in case of connection failure

File: NEWS [hunk 8]
```diff
@@ -7,11 +7,186 @@
...
+* Added put zcopy multi-rail
+* Improved logging for new protocols
+* Added system topology information
+* Added new protocols for eager offload protocols
+#### UCT
+* Extended connection establishment API
+* Added active message AM alignment in iface params
+* Added active message short IOV API.
+* Added support for interface query by operation and memory type
+* Added base address and allocted length to md_mem_query return
```
Comment (yosefe) [inline NEWS:56]: API to get allocation base address and length

File: NEWS [hunk 9]
```diff
@@ -7,11 +7,186 @@
...
+* Added md_dereg_v2 API
+#### UCS
+* Added log filter by source file name.
+* Added checking for last element in fraglist queue
+* Added a method to get IP address from sockaddr.
+* Added memory usage limits to registration cache
+#### UCM
+* Improved x86 parser to recognize some mov flavors
+* Added check for ElfW macro
```
Comment (yosefe) [inline NEWS:65]: it's minor, can remove

File: NEWS [hunk 10]
```diff
@@ -7,11 +7,186 @@
...
+#### CUDA
+* Added registration for whole CUDA allocations
+* Added CUDA-IPC keepalive
+* Adjusted performance estimations
+* Added Improve logging
+* Added allocation methods for CUDA pinned/managed memory
+* Added support for a global  cuda_ipc cache
```
Comment (yosefe) [inline NEWS:72]: extra space before cuda_ipc

File: NEWS [hunk 11]
```diff
@@ -7,11 +7,186 @@
...
+#### RDMA CORE (IB, ROCE, etc.)
+* Added remote QP info for tx error CQEs
```
Comment (yosefe) [inline NEWS:74]: Print QP info in case of completion with error

File: NEWS [hunk 12]
```diff
@@ -7,11 +7,186 @@
...
+* Adjusted performance estimations
+* Added Improve logging
+* Added allocation methods for CUDA pinned/managed memory
+* Added support for a global  cuda_ipc cache
+#### RDMA CORE (IB, ROCE, etc.)
+* Added remote QP info for tx error CQEs 
+* Refactored of FC send operations
+* Added support for  DevX unique QPN allocation
+* Optimized endpoint lookup for  DCI
+* Added support for fetching device/vendor ids based on parent PCIe
```
Comment (yosefe) [inline NEWS:78]: Add support for RDMA sub-function (SF)

File: NEWS [hunk 13]
```diff
@@ -7,11 +7,186 @@
...
+* Added support for DCI via DEVX
+* Added DCI pool per LAG port
+* Added support  for RoCE IP reachability check using a subnet mask
+* Added active message short IOV for UD/DC/RC mlx, UD/RC verbs
+* Added endpoint keep alive check for UD
+* Suppressed warning if device can't be opened
+* Added support for multiple flush cancel without completion
+* Added ignore for  devices with invalid GID
```
Comment (yosefe) [inline NEWS:86]: extra space before devices

File: NEWS [hunk 14]
```diff
@@ -7,11 +7,186 @@
...
+### Bugfixes:
+#### Build
+* Fixes in libnuma detection macro
+* Fixes for cross compilation support
+* Fixes for --without-dc compilation
+#### Continues Integration
+* Fixes in Azure pipeline build system
+* Fixes in Coverity CI
+#### Packaging
+* Fixes in RPM installation script - remove generation of default ucx.conf
```
Comment (yosefe) [inline NEWS:119]: can remove since it was added and removed in this release

File: NEWS [hunk 15]
```diff
@@ -7,11 +7,186 @@
...
+* Fixes addressing deadlock between DCI resources and RDMA_READ credits
+* Fixes in DSCP for RoCE DCT
+* Fixes in flush(cancel) flow
+* Fixes preventing segfault in uct_rdmacm_cm_ep_str
+* Fixes in scatter-gather entries logging
+* Fixes for compilation with experimental verbs
+* Fixes in UD dgid filtering
+* Fixes in domain resources destroying
+* Fixes in PCIe bandwidth calculation
+* Fixes addressing  CQ creation failure using legacy ibv API
```
Comment (yosefe) [inline NEWS:162]: extra space

**Comment by shamisp** [comment]
@yosefe - fixed

**Review by yosefe — APPROVED** [review]

**Pull request merged**